pkgs/unified_analytics/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Remove the `pddFlag` now that the revisions to the PDD have been finalized to persist data in the log file and session json file
 - Opting out will now delete the contents of the CLIENT ID, session json, and log files; opting back in will regenerate them as events send
 - `enableAsserts` parameter added to constructors for `Analytics` to check body of POST request for Google Analytics 4 limitations
+- Now checking if write permissions are enabled for user's home directory, if not allowed, `NoOpAnalytics` returned by `Analytics` factory constructor
```

Comment:
was 1.1.1 already published?

Reply:
It will be once this PR is merged into master, I made the mistake of not adding the suffix. In the future, I will use separate feature branches or have the suffix at HEAD for master until it is ready to get published

Reply:
Ok, sounds good. I would recommend using `# NEXT` in the changelog before you're actually ready to publish, unless you know there's a breaking change that would necessitate a major version bump.

Speaking of which, should the next version be 2.0? https://github.com/dart-lang/tools/commit/b55f0d47a99b1465996aeabc3f272dc653ef0573#diff-e07cea1b47833ed9526208c8d5cf51ea10672e065ca4aa9213a4cd19d5b8d7bf looks breaking.

Reply:
Yeah I suppose that is true, the reason we had those overrides at first in the default constructor was so that I could override them in the `test/pdd_approved_test.dart`. I'm also certain that only the analysis server and flutter cli are the only users of this package and I know they didn't provide overrides.

But I will go ahead and make the version `2.0.0` since it definitely would be a breaking change